services/voxtral-local/tests/test_endpoints.py

```
@@ -197,3 +197,165 @@ def test_extracts_speech_dictionary(self):
+        )
+
+        transcription_instruction = "\n\n".join(instruction_parts)
+
+        # Should use generic "Transcribe" without language
+        assert "Transcribe the following audio." in transcription_instruction
+        assert "in auto" not in transcription_instruction
+
+    def test_grammar_instruction_is_language_agnostic(self):
+        """Test that grammar instruction works for all languages, not just English."""
+        context = None
+        language = None
```

Ruff (0.14.13)

Local variable `language` is assigned to but never used

Remove assignment to unused variable `language`

(F841)